anthos-bm-utils/abm-nettest/README.md

```diff
@@ -0,0 +1,71 @@
+## What is this?
+
+ABM nettest enables users/support members to find any connectivity issues
+among k8s pods/nodes/svcs/external targets (except for connections from external
+targets to pods/nodes/svcs). This playbook will help to quickly spot any
+connectivity issues among k8s components in the cluster of interest.
+
+## Quickstart
+
+### Run Nettest
```

Comment:
It might be useful to show how to check if the nettest pod has completed, so the users know they can check the logs at that point.

Reply:
See my attempt to capture at "Get the test result" section